.ci/upload.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+rm -rf dist
+mkdir dist/
+
+export STORAGE_ACCOUNT_NAME=platopslackhelpbotai
+export CONTAINER_NAME=the-hmcts-way
+
+function extractMain () {
+    local path=$1
+
+    DIRECTORY=$(dirname "$path")
+    mkdir -p dist/$DIRECTORY
+
+    cat $path | htmlq --text 'main' --remove-nodes '[data-module="page-expiry"]' > dist/$path
```

Comment:
extract only the main content so we aren't getting the header and side bar.
Also drop the page expiry section.

I originally was using the Azure Blob Storage default behaviour of stripping html when importing to Azure AI search but it dropped working when I extracted `main`, as it requires you to at least be inside an `html` block.

Instead of re-adding it I just used the `htmlq` functionality (`--text`) for it which seems to work nicely